packages/server/dataloader/jiraServerLoaders.ts

```diff
@@ -14,6 +36,47 @@ export type JiraServerProject = JiraServerRestProject & {
   providerId: number
 }
 
+export const jiraServerIssue = (
+  parent: RootDataLoader
+): DataLoader<JiraServerIssueKey, JiraServerIssue | null, string> => {
+  return new DataLoader<JiraServerIssueKey, JiraServerIssue | null, string>(
+    async (keys) => {
+      const results = await Promise.allSettled(
+        keys.map(async ({teamId, userId, issueId}) => {
+          const manager = await TaskIntegrationManagerFactory.initManager(parent, 'jiraServer', {
```

Comment:
-1 I think here we should create `JiraTaskIntegrationManager` directly. This also avoids dataloader calls manager which calls dataloader scenario.